explainer.md

```diff
@@ -139,6 +139,11 @@ navigator.registerContentHandler(
 
 Presumably this API provided readonly access to the file.
 
+##### Why not resurrect this now more vendors are interested?
+1. It requires file handles to be encodable as a string (this will not be possible with the [native-file-system](https://github.com/WICG/native-file-system/blob/master/EXPLAINER.md) API, which is required for writable files).
```

Comment:
See example 17 in https://www.w3.org/TR/html52/webappapis.html#custom-scheme-and-content-handlers-the-registerprotocolhandler-and-registercontenthandler-methods

I would start the section with something like "registerContentHandler was not designed to handle files being opened, rather it was designed to handle link clicks that resolve to custom mime types (see <above link>). "

And then follow with 1 and 2 below.